pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/documentdb/sqlResourceSqlContainer.ts

```diff
@@ -98,7 +98,7 @@ export class SqlResourceSqlContainer extends pulumi.CustomResource {
         if (obj === undefined || obj === null) {
             return false;
         }
-        return obj['__pulumiType'] === SqlResourceSqlContainer.__pulumiType;
+        return obj['__pulumiType'] === "pulumi:providers:" + SqlResourceSqlContainer.__pulumiType;
```

Comment:
Why is this changing non-Provider resources?

Reply:
Huh good catch, well want a test checking isInstance for a non provider resource as well. I'll blame doing this quickly on Friday and not checking the codegen results before pushing up for review because I knew I needed some input on how to do the tests for it.